packages/commons/src/events/WorkqueueConfig.ts

```diff
@@ -74,6 +74,13 @@ export const WorkqueueConfig = z
   })
   .describe('Configuration for workqueue.')
 
+export const WorkqueueConfigWithoutQuery = WorkqueueConfig.pick({
+  slug: true,
+  name: true,
+  icon: true,
+  actions: true
+})
```

Comment:
```suggestion
export const WorkqueueConfigWithoutQuery = WorkqueueConfig.omit({
  query: true
})
```
Omit is just easier in this case